### PR TITLE
[FSSDK-9369] fix: fix makeRequest method of NodeRequestHandler class

### PR DESCRIPTION
**Pull request by raju-opti**
## Summary

- The response promise returned from makeRequest method of NodeRequestHandler was not resolving or rejecting for the ODP graphql api endpoint. This PR fixes the makeRequest method.

## Test plan

- All existing tests should pass

## Issues
[FSSDK-9369](https://jira.sso.episerver.net/browse/FSSDK-9369)

### CODE REVIEW

File: packages/optimizely-sdk/lib/utils/http_request_handler/node_request_handler.ts [hunk 1]
```diff
@@ -62,17 +61,14 @@ export class NodeRequestHandler implements RequestHandler {
       },
       timeout: this.timeout,
     });
-    const responsePromise = this.getResponseFromRequest(request);
+    const abortableRequest = this.getAbortableRequestFromRequest(request);
```
Comment (mikechu-optimizely) [inline packages/optimizely-sdk/lib/utils/http_request_handler/node_request_handler.ts:64]: Would this be better named `const abortableResponse = this.getAbortableResponseFromRequest(request);` ?

File: packages/optimizely-sdk/lib/utils/http_request_handler/node_request_handler.ts [hunk 2]
```diff
@@ -119,11 +115,19 @@ export class NodeRequestHandler implements RequestHandler {
    * Sends a built request handling response, errors, and events around the transmission
    * @param request Request to send
    * @private
-   * @returns Response Promise-wrapped, simplified response object
+   * @returns AbortableRequest with simplified response promise
    */
-  private getResponseFromRequest(request: http.ClientRequest): Promise<Response> {
-    return new Promise((resolve, reject) => {
+  private getAbortableRequestFromRequest(request: http.ClientRequest): AbortableRequest {
```
Comment (mikechu-optimizely) [inline packages/optimizely-sdk/lib/utils/http_request_handler/node_request_handler.ts:120]: Does anything else in the codebase use the `private getResponseFromRequest()`?  ie is it safe to rename?

**Review by mikechu-optimizely — COMMENTED** [review]
NIT comments

**Review by mikechu-optimizely — APPROVED** [review]
LGTM

**Review by opti-jnguyen — APPROVED** [review]
Looks great!

**Pull request merged**